plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -316,7 +316,7 @@ public String[] getInnerBlockPrefixes() {
 
     @Override
     public boolean isWordStart(int ch) {
-        return Character.isUnicodeIdentifierStart(ch);
+    	return Character.isUnicodeIdentifierStart(ch) || ch == '_';
```

Comment:
Please use four spaces instead of tabulation

Reply:
Thanks for the reminder, it's been modified.